### ISSUE TITLE
Workflow based on limited joypad-like input

### ISSUE THREAD

**Issue by INFU-AV** [SuperDisk/UGE#59]
TL;DR LSDj controls would be cool

Longer version: as a long time LSDj user and fan of the ergonomics it sports (being able to make whole song natively with Dpad and 4 buttons) I wanted to suggest (and if agreed, help to betatest and design) a possibility for user to operate whole tracker using only few keyboard buttons / single modern gamepad, with possible benefits:

-opening up to Native Tracker Communities with familiar workflow
-increasing accessibility for those with special needs for it
-increasing customization for those in need of specific key combos/shortcuts

Joypad based trackers bring all control that user needs while keeping amount of buttons required low, utilizing often the directional pad in multiple ways.
Currently hUGETracker allows user to transpose notes by either -+1 note or +-1 octave, so its already halfway there, but it would be very welcome to have it without complicated CTR+SHIFT combo but enable editing values while holding one button, Left/Right adjusting 1 note up or down, and Up/Down adjusting octaves in same manner.
I could go on talking about possibilities but I will just end this message by saying that its entirely possible for hUGETracker to be usable while having only 4 Directional + 4 other buttons used for all various editing/selecting/erasing/pasting things users do, including changing screens/instruments/exporting!

**Comment by SuperDisk** [comment]
I appreciate the suggestion, but this would require a somewhat big reworking of how HT handles input. You could probably set something up using AutoHotKey or Xpadder to simulate a feature like this.
